security-analysis/security-analysis-api/src/main/java/com/powsybl/security/results/BranchResult.java

```diff
@@ -34,17 +34,27 @@
 
     private final double i2;
 
+    /**
+     * Flow transfer from the branch in contingency to the branch with id branchId.
```

Comment:
Can you define the flow transfer in a different way such as: 
Note p1i,N the active power flow on side 1 of the branch branchId, and p1i, N-1 the active power flow on side 1 of the same branch. Note p1j,N the active power flow on side 1 of a branch j. After the lost of this branch, we have:
p1,N-1 = P1, N + flow transfer(j->i) * p1j,N

Reply:
For the last part did you rather mean p1i,N = P1i, N-1 + flow transfer(j->i) * p1j,N-1 ?